assets/js/charts.js

```diff
@@ -87,11 +87,13 @@ var charts = (function() {
     });
 
     let chart = document.createElementNS(SVG_NS, 'svg');
+
     chart.classList.add('percentAreaChart');
-    chart.setAttribute('width', '100%');
+    chart.setAttribute('width', '50vw');
```

Comment:
Can this be changed to 100% width + some max-width? It would be better to use all of the space until it reaches the ideal max